scripts/service-status.py

```diff
@@ -53,22 +67,38 @@ def main():
             service.enabled = nameValuePair[1] == "enabled" or nameValuePair[1] == "static"
 
     # Verify the service states CC expects
-    dnsmasqInstalled = dockerInstalled = ntpInstalled = servicedInstalled = False
+    dnsmasqInstalled = dockerInstalled = ntpInstalled = servicedInstalled = corosyncInstalled = pacemakerInstalled = False
     for service in services:
         if service.name.startswith("dnsmasq"):
             dnsmasqInstalled = True
             service.validateEnabledAndActive()
-        elif service.name.startswith("docker"):
-            dockerInstalled = True
-            service.validateEnabledAndActive()
         elif service.name.startswith("firewalld"):
             service.validateDisabledAndStopped()
         elif service.name.startswith("ntp"):
             ntpInstalled = True
             service.validateEnabledAndActive()
+        elif not isHA:
+            if service.name.startswith("docker"):
+                dockerInstalled = True
+                service.validateEnabledAndActive()
+            elif service.name.startswith("serviced"):
+                servicedInstalled = True
+                service.validateEnabledAndActive()
+        #
+        # Every check below assumes we're in an HA environment.
+        elif service.name.startswith("corosync"):
+            corosyncInstalled = True
+            service.validateEnabledAndActive()
+        elif service.name.startswith("pacemaker"):
+            pacemakerInstalled = True
+            service.validateEnabledAndActive()
+        elif service.name.startswith("docker"):
+            dockerInstalled = True
+            service.validateDisabledAndStopped()
         elif service.name.startswith("serviced"):
             servicedInstalled = True
-            service.validateEnabledAndActive()
+            service.validateDisabledAndStopped()
+
```

Comment:
I'm a little confused here - are docker and serviced disabled and stopped on both nodes in an HA environment?